lib/analyzer/static-analyzer.ts

```diff
@@ -200,10 +204,11 @@ export async function analyze(
     );
     const poetryDependenciesScanResults = await poetryFilesToScannedProjects(
       getFileContent(extractedLayers, getPoetryAppFileContentAction.actionName),
+      collectApplicationFiles,
```

Comment:
Can't we collect the application files separately from poetry/pip? That could simplify the code and reduce the number of changes

Reply:
Done